extensions/omni_python/docs/intro.md

```diff
@@ -0,0 +1,196 @@
+# Intro
+
+`omni_python` allows you to seamlessly integrate Python code within Omnigres, which are then used as stored procedures in the database. `omni_python` also allows you to integrate Flask framework and serve HTTP requests from directly within Omnigres.
+
+## Prerequisites
+
+Create `omni_python` extension if it is not installed.
+```postgresql
+create extension if not exists omni_python cascade;
+```
+
+We would also need some helper extensions for loading relevant files from filesystem.
+```postgresql
+create extension if not exists omni_schema cascade;
+create extension if not exists omni_vfs cascade;
+```
+
+## Getting started
+
+1. Create a directory on your host system containing the Python files, and corresponding `requirements.txt`, and mount the volume to the Docker container.
+
+2. We need to add the following to `requirements.txt` file to support Python integration:
+```shell
+omni_python
+```
+
+3. Create Python files. You can create more than one files, as long as they are in the directory that you would mount as a volume inside the Docker container for running Omnigres.
+
+Create functions in Python as you would, and annotate them with `@pg` to make sure they are loaded into the database.
+
+For example, let's make two Python files.
+
+`hello.py`:
+```python
+from omni_python import pg
+
+@pg
+def hello() -> str:
+    return "Hey there!"
+```
+
+`maths.py`:
+```python
+from omni_python import pg
+
+@pg
+def add(a: int, b: int) -> int:
+    return a + b
+
+@pg
+def subtract(a: int, b: int) -> int:
+    return a - b
+```
+
+4. Create a function to define an identifier for the local directory you want to load into the database. For the example, the local directory is `python-files`.
+
+```postgresql
+create or replace function demo_function() returns omni_vfs.local_fs language sql
+as $$
+select omni_vfs.local_fs('/python-files')
```

Comment:
Just a note: the upcoming CLI tooling will take care of directory mapping.

Not sure we need to do anything about this here, but I was thinking maybe we can leave a tip there (there`s tip admonition syntax in mkdocs: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:tip). Thoughts on whether that's useful at all?

Reply:
Have added a tip conveying this. Hope that works!